client/create-context-blocks.js

```diff
@@ -364,7 +364,8 @@ Template.create_image_section.onCreated(function(){
             width: doc.width,
             height: doc.height
           },
-          source: Session.get('newHorizontalDataSource'),
+          source: Session.get('newContextDataSource'),
+          authorId : Meteor.user()._id,
```

Comment:
is adding author id here intentional? i suspect leftover from a merge conflict. as of recently, we add it later in the chain.